lib/azure/armrest/network/network_security_group_service.rb

```diff
@@ -0,0 +1,95 @@
+module Azure
+  module Armrest
+    # Class for managing network security groups.
+    class NetworkSecurityGroupService < ArmrestService
```

Comment:
Comparing with what we have in manageiq, can we call it just `SecurityGroupService`?